temporal_capi/src/zoned_date_time.rs

```diff
@@ -37,6 +37,7 @@ pub mod ffi {
     pub struct PartialZonedDateTime<'a> {
         pub date: PartialDate<'a>,
         pub time: PartialTime,
+        pub has_utc_designator: bool,
```

Comment:
thought: We don't actually need this field in PartialZonedDateTime; since it's only `true` when it comes from OwnedPartialZonedDateTime, which we don't pass through this intermediary.


Not adding this would make the 0.0.10 -> 0.0.11 upgrade  not a breaking change in the capi, but it's a small enough fix that it doesn't matter too much. Open to doing it either way, thoughts?

Reply:
I went ahead and removed this field from FFI. This change is no longer FFI-breaking.